### PR TITLE
Add regression test for bug 1317819 - webconsole logging

### PR DESCRIPTION
**Pull request by bgrins**
### Summary of Changes

* Copied in test change from https://bugzilla.mozilla.org/show_bug.cgi?id=1317485#c6, confirmed was passing at https://treeherder.mozilla.org/#/jobs?repo=try&revision=1abffb9a66feb8a22dd64b205701dcdd6144db74 but probably should re-run after Bug 1317819
* Depends on https://bugzilla.mozilla.org/show_bug.cgi?id=1317819 landing

### CODE REVIEW

File: src/test/mochitest/head.js [hunk 1]
```diff
@@ -301,7 +301,8 @@ function createDebuggerContext(toolbox) {
  */
 function initDebugger(url, ...sources) {
   return Task.spawn(function* () {
-    const toolbox = yield openNewTabAndToolbox(EXAMPLE_URL + url, "jsdebugger");
+    url = url.startsWith("data:") ? url : EXAMPLE_URL + url;
```
Comment (jasonLaster) [inline src/test/mochitest/head.js]: 👍

**Comment by jasonLaster** [comment]
bumped w/ the frame fix

**Comment by jasonLaster** [comment]
got the expected error. We can bump the docker image and see this go green.

**Comment by jasonLaster** [comment]
@jlongster re-running the tests to see if we're green now

**Comment by wldcordeiro** [comment]
This PR has been in queue for a long time and has a merge conflict now, should we close or rebase?

**Comment by clarkbw** [comment]
Just resolved the conflict, lets see how the CI run goes but this should also get another review.

**Comment by jasonLaster** [comment]
@clarkbw i think this is blocked on us updating the docker image actually

**Comment by clarkbw** [comment]
#1790 updated the docker image, lets see what happens

**Comment by jasonLaster** [comment]
hmm, worth investigating now :)

**Comment by jasonLaster** [comment]
lets rebase, must be ancient

**Comment by jasonLaster** [comment]
@clarkbw finally updated this. lets see if we can get a 🍏

**Comment by clarkbw** [comment]
Woo!

**Pull request merged**